Date: Mon, 19 Jan 2009 02:09:36 -0800 (PST)
From: shaul Karl <shaulkarl@yahoo.com>
To: Debian Bug Tracking System <submit@bugs.debian.org>
MIME-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Message-ID: <943241.69881.qm@web58505.mail.re3.yahoo.com>
Delivered-To: submit@bugs.debian.org
Resent-Sender: Debian BTS <debbugs@rietz.debian.org>
Resent-Date: Mon, 19 Jan 2009 10:12:04 +0000

Package: iptables
Version: 1.4.1.1-3
Severity: minor
Tags: patch

Index: b/iptables-restore.8
===================================================================
--- a/iptables-restore.8	2009-01-19 08:20:47.065187830 -0500
+++ b/iptables-restore.8	2009-01-19 08:22:02.739062769 -0500
@@ -33,7 +33,6 @@
 restore the values of all packet and byte counters
 .TP
 \fB\-n\fR, \fB\-\-noflush\fR 
-.TP
 don't flush the previous contents of the table. If not specified, 
 .B iptables-restore
 flushes (deletes) all previous contents of the respective IP Table.
